app/package.json

```diff
@@ -26,10 +26,13 @@
     "i18next": "^22.4.12",
     "next": "^12.1.2",
     "next-i18next": "^13.2.2",
+    "pino": "^8.19.0",
```

Comment:
I've added logging. I realize that I should've made a separate PR to keep PRs small. Hopefully this isn't a big deal. I'm also not bound a particular logging library. [Pino](https://github.com/pinojs/pino) has a great community that's very active.